tests/test_integration.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2020 D-Wave Systems Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import subprocess
+import unittest
+import time
+import os
+import sys
+
+project_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+class IntegrationTests(unittest.TestCase):
+
+    def test_sudoku(self):
+        example_file = os.path.join(project_dir, 'graph_partitioning.py')
+        output = subprocess.check_output([sys.executable, example_file])
+        output = str(output).upper()
+        if os.getenv('DEBUG_OUTPUT'):
+            print("Example output \n" + output)
+
+        with self.subTest(msg="Verify if output contains valid solution energy \n"):
+            self.assertIn("Solution energy with offset included:  97.0".upper(), output)
+        with self.subTest(msg="Verify if Counts in each partition is correct \n"):
+            self.assertIn("Counts in each partition:  [6. 6. 6. 6. 6.]".upper(), output)
+        with self.subTest(msg="Verify if Number of links between partitions \n"):
+            self.assertIn("Number of links between partitions:  97".upper(), output)
```

Comment:
Are we sure these are the only acceptable solutions? Shouldn't we use ranges? /cc @joelgdwave

Reply:
@randomir and @hemant, the solutions that I saw always seemed to be equally partitioned (the 6's) but the solution energy can vary. It can vary because it is a random graph. I would suggest not checking the energy in the testing here. The equipartitioning should be good enough.